msi/createmsi.py

```diff
@@ -114,20 +114,26 @@ def build_dist(self):
         modules = self.get_all_modules_from_dir('mesonbuild/modules')
         modules += self.get_all_modules_from_dir('mesonbuild/scripts')
         modules += self.get_more_modules()
-        modulestr = ','.join(modules)
-        python = shutil.which('python')
-        cxfreeze = os.path.join(os.path.dirname(python), "Scripts", "cxfreeze")
-        if not os.path.isfile(cxfreeze):
-            print("ERROR: This script requires cx_freeze module")
+
+        pyinstaller = shutil.which('pyinstaller')
+        if not pyinstaller:
+            print("ERROR: This script requires pyinstaller.")
             sys.exit(1)
 
-        subprocess.check_call([python,
-                               cxfreeze,
-                               '--target-dir',
-                               main_stage,
-                               '--include-modules',
-                               modulestr,
-                               'meson.py'])
+
+        pyinstaller_tmpdir = 'pyinst-tmp'
+        if os.path.exists(pyinstaller_tmpdir):
+            shutil.rmtree(pyinstaller_tmpdir)
+        pyinst_cmd = [pyinstaller,
+                      '--clean',
+                      '--distpath',
+                      pyinstaller_tmpdir,
+        ]
```

Comment:
__[Flake8]__ [E124] closing bracket does not match visual indentation ([link](https://sider.review/gh/repos/19784232/pulls/5558#issue-4278173))

<sub>posted by Sider</sub>